owncloudApp/src/main/java/com/owncloud/android/ui/activity/ToolbarActivity.kt

```diff
@@ -97,6 +101,10 @@ abstract class ToolbarActivity : BaseActivity() {
                 toolbarTitle.visibility = VISIBLE
                 false
             }
+            val textSearchView = findViewById<EditText>(androidx.appcompat.R.id.search_src_text)
+            val closeButton = searchView.findViewById<ImageView>(androidx.appcompat.R.id.search_close_btn)
```

Comment:
If this is inside the `apply` block, we don't need reference to `searchView`

Reply:
Done